src/denostore.ts

```diff
@@ -161,7 +162,7 @@ export default class Denostore {
     this.#router.post(this.#route, async (ctx: Context): Promise<void> => {
       const { response, request } = ctx;
       try {
-        const body = await request.body();
+        const body = request.body();
```

Comment:
Can we await request.body().value instead?

Reply:
resolved